todo_app/templates/index.html

```diff
@@ -10,31 +10,105 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul id="todo_list" class="list-group mb-4">
-        <!-- List the todo items here -->
-        {% for item in items %}
-        <li id="li{{ item.id}}">
-          <span>{{ item.title }} - </span>
-          <span id="status{{ item.id}}" onclick="updateStatus('{{ item.id }}', event)">{{ item.status }}</span>
-          <span onclick="deleteItem('{{ item.id }}')">(Delete)</span>
-        </li>
-        {% endfor %}
-        <li id="new_item" ><form id="new_todo_form" action="/todo/add" method="post" onsubmit="createNewItem(event)")>
-        <label for="title">New Todo:</label> <input required type="text" id="title" name="title" value="" /> <input type="submit" value="Submit" />
-      </form></li>
+      <form id="new_item_form">
+        <label for="title">New Todo:</label> <input required type="text" id="new_item_title" name="title" value="" />
+        <button type="submit" class="btn btn-primary">Create</button>
+      </form>
+
+      <h3>To Do</h3>
+      <ul id="todo_list" class="list-group">
+      </ul>
+
+
+      <h3>Doing</h3>
+      <ul id="doing_list" class="list-group">
       </ul>
+
+      <h3>Done</h3>
+      <ul id="done_list" class="list-group">
+      </ul>
+
     </div>
 
   </div>
 
   <script>
+    function newRadioButton(id, type, label, selected) {
```

Comment:
This level of complexity is why it's not normally worth writing a single page application without a framework to help you (like React or Angular)

Reply:
Yeah agreed, I think moving this to React might be a nice stretch goal for me to attempt if I get time.